lib/cli/interactive-setup/index.js

```diff
@@ -18,9 +19,14 @@ module.exports = async (context) => {
     const stepData = await step.isApplicable(context);
     if (stepData) {
       process.stdout.write('\n');
-      context.stepHistory = [];
+      context.stepHistory = new StepHistory();
       context.history.set(stepName, context.stepHistory);
       await step.run(context, stepData);
     }
   }
+  const commandUsage = Array.from(context.history.entries()).map(([step, stepHistory]) => ({
+    name: step,
+    history: stepHistory.toJSON(),
```

Comment:
Just a comment: there's no need to invoke `toJSON()`, as it's automatically invoked by internals of `JSON.stringify` when generating payload, but there's no harm in it either